index.html

```diff
@@ -2,18 +2,12 @@
 <html>
   <head>
     <title>electron es6 react boilerplate</title>
+    <script>
+      require('babel-register');
```

Comment:
Could you re-install the comment "// install babel hooks in the renderer process"? I added it because it wasn't obvious for me.